test/fixtures/parser/mysql/force_index.sql

```diff
@@ -1,8 +1 @@
-SELECT
-       onetable.f1,
-       twotable.f1
-FROM onetable
-         LEFT JOIN
-     twotable
-         FORCE INDEX FOR JOIN (idx_index)
-     ON onetable.f1 = twotable.f1
\ No newline at end of file
+SELECT * FROM onetable FORCE INDEX (idx_index);
```

Comment:
Why so many changes to this file? Could this (should this) be a new test file instead?

Reply:
I simplified this one since the original statement for the test was more complex than necessary.  I will add a new test that is specific to using the `FORCE INDEX` on a join so that the original test will still be validated.

Reply:
@barrywhart good catch, after adding the test back, my changes no longer covered it.  But I have it updated now to cover both this test and my existing tests.  Thanks!